Automattic/wp-calypso
# Stats: move countries to redux.

This branch is still in-progress and will be blocked by probably a few more forthcoming branches.

TODO:
- [x] Fix stats normalizer tests
- [x] Add `UpgradeNudge` logic back
- [x] Create selector for state-tree based CSV data - #10509
- [x] Update `<DownloadCSV/>` to use state data - #10510

@youknowriad if you have time and want to check out this branch so far, that would be great.

__To Test__
- Open a site stats day page for a site with visits data
- Note the countries module loads as expected and displays the same data as production
- Try to make sure the loading indicators also show as expected, it helps to have the API sandboxed which slows things down
- Click on the 'View All' link or the title bar to visit the Countries Summary page.
